Wandbb operator airgapped installation (#843)
wandb/docs: docs/guides/hosting/self-managed/azure-tf.md

Another deployment option uses `Redis` to cache the SQL queries and speed up the application response when loading the metrics for the experiments.

> You need to add the option `create_redis = true` to the same `main.tf` file we worked on in [Deployment Recommended](#deployment---recommended-20-mins) to enable the cache.

Reviewer: ```suggestion
You must add the option `create_redis = true` to the same `main.tf` file that you used in [recommended deployment](#deployment---recommended-20-mins) to enable the cache.
```

Deployment option 3 consists of enabling the external `message broker`. This is optional because the W&B brings embedded a broker. This option doesn't bring a performance improvement.

> The Azure resource that provides the message broker is the `Azure Event Grid`, and to enable it, you will need to add the option `use_internal_queue = false` to the same `main.tf` that we worked on the [Deployment Recommended](#deployment---recommended-20-mins)

Reviewer: ```suggestion
The Azure resource that provides the message broker is the `Azure Event Grid`, and to enable it, you must add the option `use_internal_queue = false` to the same `main.tf` that you used in the [recommended deployment](#deployment---recommended-20-mins)
```

## Other deployment options

You can combine all three deployment options adding all configurations to the same file.

> The [Terraform Module](https://github.com/wandb/terraform-azure-wandb) provides several options that can be combined along with the standard options and the minimal configuration found in [Deployment Recommended](#deployment---recommended-20-mins)

Reviewer: ```suggestion
The [Terraform Module](https://github.com/wandb/terraform-azure-wandb) provides several options that can be combined along with the standard options and the minimal configuration found in [recommended deployment](#deployment---recommended-20-mins)
```
